### ISSUE TITLE
Graphing mode: Values of Key Graph Features are not announced by Narrator

### ISSUE THREAD

**Issue by joshkoon** [microsoft/calculator#952]
**Describe the bug**
After using Analyze Equation, any Key Graph Features that have values are read by narrator as, "Edit, read-only"

**Steps To Reproduce**

1. Go to Graphing mode
2. Add an equation such as "x + 2"
3. Select "Analyze Equation" from hover buttons or context menu
4. With Narrator turned on, navigate through the list of key graph features using Capslock + left/right arrows
5. Observe Narrator announced many values as "Edit, read only"
6. With an "edit read only" control highlighted, use Capslock + down arrow
7. Observe Narrator reads the contents of the edit box.

**Expected behavior**
Narrator should read the contents of these controls when first navigating to them.

**Screenshots**

![image](https://user-images.githubusercontent.com/45607479/72634960-9036ec80-3910-11ea-875f-3605dc3938ea.png)

**Device and Application Information**
- OS Build: 10.0.19547.0
- Architecture: x64
- Application Version: 10.1912.4.0
- Region: en-US
- Dev Version Installed:

**Additional context**

**Requested Assignment**

I'm just reporting this problem.  I don't want to fix it.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by joseartrivera** [comment]
These are rich edit boxes, to get narrator to read them you can use any command that tells narrator to read the contents of the textbox, such as caps lock + down arrow. See [Appendix B: Narrator keyboard commands and touch gestures](https://support.microsoft.com/en-us/help/22806/windows-10-narrator-keyboard-commands-touch-gestures) for more examples.
